woodwork/statistics_utils/_calculate_max_dependence_for_pair.py

```diff
@@ -22,10 +22,14 @@ def _calculate_max_dependence_for_pair(result, min_shared, extra_stats):
     """
     # if pearson was not measured, mutual info must be max, since columns valid
     # for pearson are a subset of columns valid for mutual info
-    if "pearson" in result:
+    if "pearson" in result or "spearman" in result:
```

Comment:
does this mean you get spearman + pearson even if you only request one of the two?

Reply:
No, this checks that as long as we have another metric that isn't mutual info (ie pearson, spearman), we try to grab all the values to calculate the max. The `dictionary.get` method here will return `nan` if they don't exist, but otherwise will grab the values we need.